pkg/operator/ceph/object/health.go

```diff
@@ -42,19 +45,22 @@ var (
 
 // bucketChecker aggregates the mon/cluster info needed to check the health of the monitors
 type bucketChecker struct {
-	context         *clusterd.Context
-	objContext      *Context
-	interval        *time.Duration
-	port            int32
-	client          client.Client
-	namespacedName  types.NamespacedName
-	objectStoreSpec *cephv1.ObjectStoreSpec
+	context               *clusterd.Context
+	objContext            *Context
```

Comment:
This object context seems like the right struct where the creds should be stored if possible.

Reply:
Will try.